Source/Targeting.swift

```diff
@@ -181,6 +181,34 @@ import CoreLocation
         }
     }
     
+    // MARK: - External UserIds
```

Comment:
why is this entire logic needed when you already have the externalUSerID setter & getter in Prebid.shared

Reply:
Prebid.shared contains an Array of External UserIds same as what we implemented in MobileSDK whereas Targeting.shared contains setter & getter specific to each External UserId same as the way we implemented in GDPR.
So both serve different purposes.

Reply:
👍

Reply:
Done